src/internal.c

```diff
@@ -17709,9 +17709,11 @@ int SendData(WOLFSSL* ssl, const void* data, int sz)
 
         len = wolfSSL_GetMaxRecordSize(ssl, sz - sent);
 
-#ifdef WOLFSSL_DTLS
-        if (IsDtlsNotSctpMode(ssl)) {
-            len = min(len, MAX_UDP_SIZE);
```

Comment:
Just confirming this check was moved to `wolfSSL_GetMaxRecordSize`?

Reply:
Yes.